### ISSUE TITLE
Pagination Link Do Not Outputted on JSON

### ISSUE THREAD

**Issue by edwinlunando** [rails-api/active_model_serializers#1466]
Hi, I'm using `master` branch and `JSON` adapter. I want to show pagination links, this is my pagination serializer.

```ruby
# pagination collection serializer
class PaginationSerializer < ActiveModel::Serializer::CollectionSerializer

  def initialize(object, options = {})
    meta_key = options[:meta_key] || :meta
    options[meta_key] ||= {}
    options[meta_key][:pagination] = {
      current_page: object.current_page,
      next_page: object.next_page,
      prev_page: object.prev_page,
      total_pages: object.total_pages,
      total_count: object.total_count
    }
    super(object, options)
  end

end

```

This is my render code

```ruby
render json: formulas, serializer: PaginationSerializer, each_serializer: FormulaSimpleSerializer
```

There no `meta` key on the result. This happens after upgrading from 0.10.RC3 to master. Any idea?

**Comment by edwinlunando** [comment]
I tried to dig the old `ArraySerializer`

```ruby
def initialize(resources, options = {})
  @root = options[:root]
  @object = resources
  @serializers = resources.map do |resource|
    serializer_class = options.fetch(:serializer) {
      ActiveModel::Serializer.serializer_for(resource)
    }

    if serializer_class.nil?
      fail NoSerializerError, "No serializer found for resource: #{resource.inspect}"
    else
      serializer_class.new(resource, options.except(:serializer))
    end
  end
  @meta     = options[:meta]
  @meta_key = options[:meta_key]
end
```

and compare it to the new `CollectionSerializer`

```ruby
def initialize(resources, options = {})
  @root = options[:root]
  @object = resources
  @serializers = resources.map do |resource|
    serializer_context_class = options.fetch(:serializer_context_class, ActiveModel::Serializer)
    serializer_class = options.fetch(:serializer) { serializer_context_class.serializer_for(resource) }

    if serializer_class.nil?
      fail NoSerializerError, "No serializer found for resource: #{resource.inspect}"
    else
      serializer_class.new(resource, options.except(:serializer))
    end
  end
end
```

Why there is no `meta` declaration? Is this expected? What I must to do in order to add pagination links?

**Comment by edwinlunando** [comment]
Come to this rendering page, does it means we cannot use something like `PaginationSerializer` anymore? Do we need to explicitly put it in `render` method?

**Comment by edwinlunando** [comment]
Come to this [pull request](https://github.com/rails-api/active_model_serializers/pull/1233), seems the  pagination documentation on `master` branch is no longer valid. I can make a PR if you want to fix the docs?

**Comment by beauby** [comment]
@edwinlunando Yes, a docs PR would be great!

**Comment by edwinlunando** [comment]
request, pulled. https://github.com/rails-api/active_model_serializers/pull/1472

**Comment by bf4** [comment]
Closed by #1472
